docs/gemstones/git/feature_branch_workflow.md

```diff
@@ -72,11 +72,11 @@ The Feature Branch workflow is a common collaboration technique, allowing teams
 The high-level steps involved are:
    
 1. Clone the Main Repository: Directly clone the main project repository to your local machine.
-2. Create a Feature Branch: For every new task, create a new branch off the main branch with a descriptive name.
+2. Create a Feature Branch: For every new task, create a new branch of the main branch with a descriptive name.
```

Comment:
Not sure here between off and of. It means the new branch is a copy from the main branch.

Reply:
Correct as is.  

Can also be expressed like:

"Start a new offshoot from the main line for each task, and give it a name that tells you what the task is about.